flink-runtime/src/main/java/org/apache/flink/runtime/fs/hdfs/HadoopDataInputStream.java

```diff
@@ -89,4 +99,14 @@ public long skip(long n) throws IOException {
 	public org.apache.hadoop.fs.FSDataInputStream getHadoopInputStream() {
 		return fsDataInputStream;
 	}
+
+	public void forceSeek(long seekPos) throws IOException {
```

Comment:
I agree that doc wouldn't hurt. This class as a whole was rather undocumented, but it is also internal and user will only interact through`FSDataInputStream`, which is not exposing those methods. Can write something anyways :)